### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by MaksGovor**

### CODE REVIEW

File: src/components/navigation/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Container, Typography, Link, Divider, Avatar, Popover } from '@mui/material';
+import {
+	NavToolbar,
+	FlexBox,
+	InlineBlock,
+	MBreadcrumbs,
+	PopoverContainer,
+	PopoverButton,
+} from './Navigation.styles';
```
Comment (tedi4t) [inline src/components/navigation/Navigation.tsx:10]: Maybe move it to the bottom and @mui dependencies upper?

File: src/components/navigation/Navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	const handleClose = (): void => setOpened(false);
+
+	return (
+		<>
+			<NavToolbar>
+				<Container maxWidth={false}>
+					<FlexBox>
+						<FlexBox>
+							<Link
+								underline='none'
```
Comment (tedi4t) [inline src/components/navigation/Navigation.tsx:38]: Can we move this styles to *.styles.ts? I think that we should keep only logic and very small quantity of styles in this file.

File: src/components/navigation/Navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+								<Typography variant='h4' display='inline-block'>
+									ICS
+								</Typography>
+							</Link>
+
+							<Divider orientation='vertical' variant='middle' flexItem sx={{ ml: 2 }} />
+							<MBreadcrumbs aria-label='breadcrumb'>
+								<Link
+									underline='hover'
+									sx={{ display: 'flex', alignItems: 'center' }}
```
Comment (tedi4t) [inline src/components/navigation/Navigation.tsx:53]: Same with this styles

**Comment by MaksGovor** [comment]
All fixed

**Pull request merged**